spark/src/main/scala/org/apache/spark/sql/delta/util/DeltaCommitFileProvider.scala

```diff
@@ -60,6 +60,17 @@ case class DeltaCommitFileProvider(
       case _ => FileNames.unsafeDeltaFile(resolvedPath, version)
     }
   }
+
+  def listSortedUnbackfilledDeltaFiles(startVersionOpt: Option[Long] = None): Seq[(Long, Path)] = {
```

Comment:
Can you add a short docstring to at least indicate that this not do any additional IO and only returns what it already knows?